chapters/api_concepts.txt

```diff
@@ -573,6 +573,11 @@ specified before unmapping to avoid populating the array with undefined values.
 of all objects in the array. Reference counts are updated on creation
 of the array object and when unmapping the array.
 
+Use of the arrays inside ANARI may cause accesses at indices derived
+from parameters or other arrays (for example when drawing an indexed geometry).
+If these accesses would cause out of bounds reads the read value is undefined.
+However out of bounds accesses must not result in abnormal program termination.
+
```

Comment:
See https://registry.khronos.org/OpenGL/extensions/KHR/KHR_robustness.txt for inspiration:
> Variables of the same type ... Undefined behavior results from indexing
an array with a non-constant expression that is greater than or equal to
the array size or less than 0. If robust buffer access is enabled (see
section 10.3.1.1rob of the OpenGL ES 3.1 API Specification), **such
indexing must not result in abnormal program termination. The results
are still undefined, but implementations are encouraged to produce zero
values for such accesses.** Arrays only have a single dimension ...